### PR TITLE
[WIP] Muf proxy

### PR DESCRIPTION
**Pull request by pivnicek**

### CODE REVIEW

File: Controller/DataPermissionsController.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	/**
+	 * List user's dataPermissions
+	 * @Route("/{config}/dataPermissions/{email}")
+	 * @Method({"GET"})
+	 */
+	public function listDataPermissionsAction($config, $email)
+	{
+		$this->setConfiguration($config);
+
+		$user = $this->getDatabase()->getUsers()->get($email);
```
Comment (JakubMatejka) [inline Controller/DataPermissionsController.php]: I think it is not necessary, we can assign filters to any user as this is just a proxy. You just will have to get uid of the user. Probably the best way (altough not optimal ;os ) is to list users from project and find the one you need: https://github.com/keboola/gooddata-php-client/blob/master/src/Keboola/GoodData/Projects.php#L101 (do you know how to work with yields, right? You will need two nested foreachs, the method will give you array of arrays)

Comment (pivnicek) [inline Controller/DataPermissionsController.php]: jj,  like you have here: 
https://github.com/keboola/gooddata-writer/blob/master/Task/AssignFiltersToUser.php#L66
ok, I'll change it over, thanks

Comment (pivnicek) [inline Controller/DataPermissionsController.php]: aha, I understand your point now, yes, there's no need to involve the db, good catch, thanks

File: Controller/DataPermissionsController.php [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	/**
+	 * Assign filters to user
+	 * @Route("/{config}/dataPermissions/{email}")
+	 * @Method({"PUT"})
+	 */
+	public function assignDataPermissionAction($config, $email)
+	{
+		$this->setConfiguration($config);
+
+		$user = $this->getDatabase()->getUsers()->get($email);
```
Comment (JakubMatejka) [inline Controller/DataPermissionsController.php]: the same as above

File: Task/deleteDataPermission.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<?php
```
Comment (JakubMatejka) [inline Task/deleteDataPermission.php]: The filename starts on lowercase, I'm not sure if it would work in any case

Comment (pivnicek) [inline Task/deleteDataPermission.php]: huh, that's weird, I changed it locally (yes, it didn't work as lowercase ofc), but it's not reflected here, hmmm...  mrknu na to

Comment (JakubMatejka) [inline Task/deleteDataPermission.php]: I think git has sometimes problem with changes but `git mv` should do it (http://stackoverflow.com/a/24979063/436706)

File: Controller/DataPermissionsController.php (hunk 1 above)
Comment (pivnicek) [inline Controller/DataPermissionsController.php]: @JakubMatejka I noticed that maybe here https://github.com/keboola/gooddata-writer/blob/master/Task/AssignFiltersToUser.php#L66 there could be better to have `break 2` to break both loops when uid is found.  Shall I change it?

**Pull request merged**